packages/core/src/browser/shell/tab-bar-toolbar.tsx

```diff
@@ -129,12 +129,30 @@ export class TabBarToolbar extends ReactWidget {
             classNames.push(iconClass);
         }
         const tooltip = item.tooltip || (command && command.label);
-        return <div key={item.id} className={`${TabBarToolbar.Styles.TAB_BAR_TOOLBAR_ITEM}${command && this.commandIsEnabled(command.id) ? ' enabled' : ''}`}
-            onMouseDown={this.onMouseDownEvent} onMouseUp={this.onMouseUpEvent} onMouseOut={this.onMouseUpEvent} >
-            <div id={item.id} className={classNames.join(' ')} onClick={this.executeCommand} title={tooltip}>{innerText}</div>
+        const toolbarItemClassNames = command ? this.getToolbarItemClassNames(command.id) : '';
+        return <div key={item.id}
+            className={`${TabBarToolbar.Styles.TAB_BAR_TOOLBAR_ITEM} ${toolbarItemClassNames}`}
```

Comment:
I think I'd be inclined to put the constant classname `...TAB_BAR_TOOLBAR_ITEM` in the `getToolbarItemClassNames` function as well, as is done in for the tree node class names:

https://github.com/eclipse-theia/theia/blob/b6add4a6e6e97cbdff73279fab0a0b116576ccb5/packages/core/src/browser/tree/tree-widget.tsx#L881-L896